examples/ex9p.cpp

```diff
@@ -64,31 +64,100 @@ double inflow_function(const Vector &x);
 // Mesh bounding box
 Vector bb_min, bb_max;
 
+// Type of preconditioner for implicit time integrator
+enum class PrecType : int
+{
+   ILU = 0,
+   AIR = 1
+};
+
+#if MFEM_HYPRE_VERSION >= 21800
```

Comment:
Although AIR AMG is documented in hypre.hpp, some brief documentation here should be added here, either to describe AIR or to reference the documentation in hypre.hpp. Since this is an example, it needs to be clear for new users.